query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/SourceValidationUtils.java

```diff
@@ -84,15 +83,15 @@ public Report duplicateSourceNameExists(StringField sourceName) {
      * @return a {@link Report} containing an {@link org.codice.ddf.admin.api.report.ErrorMessage}s on failure.
      */
     public Report duplicateSourceNameExists(StringField sourceName, PidField pid) {
-        ReportImpl sourceNameReport = new ReportImpl();
+        Report sourceNameReport = Reports.emptyReport();
         if (pid.getValue() != null) {
             sourceNameReport = serviceCommons.serviceConfigurationExists(pid);
-            if (!sourceNameReport.containsErrorMsgs() && !findSourceNameMatch(pid.getValue(),
+            if (!sourceNameReport.containsErrorMessages() && !findSourceNameMatch(pid.getValue(),
                     sourceName.getValue())) {
-                sourceNameReport.addMessages(duplicateSourceNameExists(sourceName));
+                sourceNameReport.addErrorMessages(duplicateSourceNameExists(sourceName).getErrorMessages());
```

Comment:
Should we have an `addErrorMessages` in the report interface that can take in a report instead? It seems like everyone is calling `getErrorMessages` on their own report and passing these values in